src/EventStore.Core/Services/Transport/Http/Controllers/AdminController.cs

```diff
@@ -41,16 +44,74 @@ private void OnPostShutdown(HttpEntityManager entity, UriTemplateMatch match)
 
         private void OnPostScavenge(HttpEntityManager entity, UriTemplateMatch match)
         {
-            if (entity.User != null && (entity.User.IsInRole(SystemRoles.Admins) || entity.User.IsInRole(SystemRoles.Operations)))
-            {
-                Log.Info("Request scavenging because /admin/scavenge request has been received.");
-                Publish(new ClientMessage.ScavengeDatabase(new NoopEnvelope(), Guid.Empty, entity.User));
-                entity.ReplyStatus(HttpStatusCode.OK, "OK", LogReplyError);
-            }
-            else
+            int startFromChunk = 0;
+
+            var startFromChunkVariable = match.BoundVariables["startFromChunk"];
```

Comment:
Should be `QueryParameters` instead of `BoundVariables`

Reply:
This works fine. The variable is bound due to the template.

Reply:
Didn’t work when I tested it. They got completely ignored